jobs/env/pull-kubernetes-e2e-kubeadm-gce.env

```diff
@@ -1,10 +0,0 @@
-### job-env
-
-
-GINKGO_PARALLEL=y
```

Comment:
this is not a comment :-)

you can add a `--ginkgo-parallel=30` to config.json instead

Reply:
The pull job is inactivated and not used.
Let's check later today if it makes sense to enable ginkgo in parallel on Slack...
If it makes sense, I'll send a PR to do that for all kubeadm jobs :)

Reply:
okay - you just need to remember :-)